test/TestVariableHasSpaces.py

```diff
@@ -11,11 +11,16 @@
 - name: good variable format
   debug:
     msg: "Value: {{ good_format }}"
-- name: jijna escaping allowed
+- name: jinja escaping allowed
   debug:
     msg: "{{ '{{' }}"
-- name: jijna escaping allowed
+- name: jinja escaping allowed
   shell: docker info --format '{{ '{{' }}json .Swarm.LocalNodeState{{ '}}' }}' | tr -d '"'
+- name: jinja whitespace control allowed
+  debug:
+    msg: |
+      {{ good_format }}/
+      {{- good_format }}
```

Comment:
extend this test with @ssbarnea's one

Reply:
@netzvieh I hope you can make the small fix that was requested as I want to help merging this bugfix.

Reply:
Added.